os/board/rtl8730e/src/rtl8730e_boot.c

```diff
@@ -242,6 +242,7 @@ void board_gpio_initialize(void)
 				{PA_23, PIN_INPUT, PullNone},
 				/* PB_20 is gpio pin number for LED */
 				{PB_20, PIN_OUTPUT, PullDown},
+				{PB_22, PIN_OUTPUT, PullNone},
```

Comment:
os/arch/arm/src/amebasmart/amebasmart_i2s.c
	.i2s_mclk_pin = PB_22,

PB22 pin is setting I2S2 mclk. 
We need to check first if we can change I2S2 mclk to un-used pin

Reply:
rtk will raise PR about I2S2,
@ritesh55555 Let's merge your PR, Please update

Reply:
done